meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/start-resin-supervisor

```diff
@@ -7,6 +7,14 @@ SUPERVISOR_IMAGE_ID=$(balena inspect --format='{{.Id}}' "$SUPERVISOR_IMAGE:$SUPE
 SUPERVISOR_CONTAINER_IMAGE_ID=$(balena inspect --format='{{.Image}}' resin_supervisor || echo "")
 
 runSupervisor() {
+    # If self-signed root CA exists, add the CA for the specified registry
+    BALENA_DECODED_ROOT_CA=$(if [ ! -z "${BALENA_ROOT_CA}" ]; then echo "${BALENA_ROOT_CA}" | base64 -d -w0; fi)
+    if [ ! -z "${BALENA_DECODED_ROOT_CA}" ]; then
+        CERT_DIR="/mnt/state/root-overlay/etc/docker/certs.d/${REGISTRY_ENDPOINT}"
```

Comment:
One more thing here. The root overlay directory was only meant for entries that are bind mounted to rootfs. This is not the case for this specific change. As well the root-overlay has a reset mechanism which might blow its content (https://github.com/resin-os/meta-resin/blob/master/meta-resin-common/recipes-support/resin-state-reset/resin-state-reset/resin-state-reset). So currently I would recommend using another path.